spec/features/07_user_rates_figure_spec.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+require 'spec_helper'
+require 'rails_helper'
+
+# [] user can adda rating
+# [] rating should be added to average rating
+# [] user should be able to optionally add review to rating
+
+feature 'user adds rating for historical figure' do
+  let!(:user) { FactoryGirl.create(:user) }
+  let!(:figure) { FactoryGirl.create(:figure) }
+  def sign_in(user)
+    visit root_path
+    click_link 'Sign In'
+    fill_in 'Email', with: user.email
+    fill_in 'Password', with: user.password
+    click_button 'Log in'
```

Comment:
you can move this sign_in into a helper file so that it is accessible to other tests as well such as in the spec helper